src/Controller/EventController.php

```diff
@@ -66,4 +97,50 @@ public function showEvent(Event $event)
     {
         return ['event'=>$event];
     }
+
+    /**
+     * @Route("/event/edit/{id}/{token}", name="edit_event", requirements={"id"="\d+"})
+     * @Template("/admin/event/edit.html.twig")
+     */
+    public function editEvent(Request $request, Event $event, $token, FileUploader $fileUploader)
+    {
+        if ($event->getToken() != $token){
+            throw $this->createNotFoundException('Возможно, вы пытаетесь отредактировать ранее созданное событие, но это неправильная ссылка. 
+            В таком случае обратитесь к администратору.');
+        } else {
```

Comment:
@daria-sieroshtan 
You never need `if`-`else` when you have `throw` in `if`
Just do
```PHP
if ($condition) {
    throw \Exception();
}
// more code here without else
```